felt/src/lib.rs

```diff
@@ -182,10 +182,10 @@ mod test {
         #[test]
         // Property-based test that ensures, for 100 {x} and {y} values that are randomly generated each time tests are run, that the result of the division of {x} by {y} is the inverse multiplicative of {x} --that is, multiplying the result by {y} returns the original number {x}. The values of {x} and {y} can be either [0] or a very large number.
         fn div_is_mul_inv(ref x in "(0|[1-9][0-9]*)", ref y in "[1-9][0-9]*") {
-            prop_assume!("0" != y);
 
             let x = &Felt::parse_bytes(x.as_bytes(), 10).unwrap();
             let y = &Felt::parse_bytes(y.as_bytes(), 10).unwrap();
+            prop_assume!(&Felt::zero() != y);
```

Comment:
Sorry, when we talked I forgot there was this more efficient way:
```suggestion
            prop_assume!(!y.is_zero());
```